conversational-skills/procode-skill-sdk-js/src/mock-server/app.js

```diff
@@ -0,0 +1,75 @@
+// import dependencies and initialize express
+
+import express from 'express';
+import bodyParser from 'body-parser';
+
+import {createBluepointsSkill} from '../bluepoints/BluepointsSkillFactory.js';
+
+const app = express();
+
+
+
+// enable parsing of http request body
+app.use(bodyParser.urlencoded({extended: true}));
+app.use(bodyParser.json());
+
+
+// routes and api calls
+
+const bluepointsSkill = await createBluepointsSkill('en'); 
+
+/**
+ * This function lists the skills available by your provider. It will be called when you select which skills are part of your Assistant.
+ * It follows the OAS specification /orchestrate Operation. See https://github.com/watson-developer-cloud/assistant-toolkit/blob/master/conversational-skills/procode-endpoints.md#oas
+ * @param req - This is an express request. The request contains the following info `provider_id` ( path param), assistant_id ( query param), environment_id ( query param)
```

Comment:
```suggestion
 * @param req - This is an express request. The request contains the following info `provider_id` ( path param ), assistant_id ( query param ), environment_id ( query param )
```

a spacing nit on the above